OSDOCS-4320: Recommended host practices and Recommended cluster scaling practices  section reviews (#53348)
openshift/openshift-docs: modules/master-node-sizing.adoc

| 252
| 4000
| 24
| 64, but 128 if using `OVNKubernetes`

| 501
| 4000
| 16
| 96, but untested with `OVNKubernetes`

|===

> On a large and dense cluster with three control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted, or fails. The failures can be due to unexpected issues with power, network, underlying infrastructure, or intentional cases where the cluster is restarted after shutting it down to save costs. The remaining two control plane nodes must handle the load in order to be highly available, which leads to increase in the resource usage. This is also expected during upgrades because the control planes are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall CPU and memory resource usage on the control plane nodes to at most 60% of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.

Reviewer: ```suggestion
On a large and dense cluster with three control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted, or fails. The failures can be due to unexpected issues with power, network, underlying infrastructure, or intentional cases where the cluster is restarted after shutting it down to save costs. The remaining two control plane nodes must handle the load in order to be highly available, which leads to increase in the resource usage. This is also expected during upgrades because the control plane nodes are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall CPU and memory resource usage on the control plane nodes to at most 60% of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.
```

Author: Thanks @rsevilla87! Suggestion applied.

> On a large and dense cluster with three control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted, or fails. The failures can be due to unexpected issues with power, network, underlying infrastructure, or intentional cases where the cluster is restarted after shutting it down to save costs. The remaining two control plane nodes must handle the load in order to be highly available, which leads to increase in the resource usage. This is also expected during upgrades because the control plane nodes are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall CPU and memory resource usage on the control plane nodes to at most 60% of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.

Reviewer: I'd like to add more detail of the test:

```suggestion
The numbers from the table above have been taken from an OpenShift AWS IPI deployed cluster, using r5.4xlarge and m5.2xlarge instances as control-plane and worker nodes respectively.

On a large and dense cluster with three control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted, or fails. The failures can be due to unexpected issues with power, network, underlying infrastructure, or intentional cases like when the cluster is restarted after shutting it down to save costs. The remaining two control plane nodes must handle the load in order to be highly available, which leads to increase in the resource usage. This is also expected during upgrades because the control plane nodes are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall CPU and memory resource usage on the control plane nodes to at most 60% of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.
```

Author: @rsevilla87 Added. Thanks!

| 252
| 4000

> | 16

Reviewer: Hey @ahardin-rh , can we keep 16 here and add  "24 if using `OVNKubernetes`?"

> | 24

Reviewer: ```suggestion
| 24, but 24 if using `OVNKubernetes`
```

| 64, but 128 if using `OVNKubernetes`

| 501
| 4000

> | 16

Reviewer: ```suggestion
| 16 but 24 if using `OVNKubernetes`
```

> | 96, but untested with `OVNKubernetes`

Reviewer: I think is better to move the OVN statement to column with the number of worker nodes as I suggested

|===

> The data from the table above is based on an {product-title} installer-provisioned infrastructure (IPI) AWS cluster, using r5.4xlarge instances as control-plane nodes and m5.2xlarge instances as worker nodes.

Reviewer: ```suggestion
The data from the table above is based on an {product-title} running on top of AWS, using r5.4xlarge instances as control-plane nodes and m5.2xlarge instances as worker nodes.
```

Author: Updated. Thanks!
